lib/include/up-client-zenoh-cpp/rpc/zenohRpcClient.h

```diff
@@ -85,7 +86,8 @@ class ZenohRpcClient : public RpcClient {
         std::shared_ptr<ThreadPool> threadPool_;
 
         static constexpr auto requestTimeoutMs_ = 5000;
-        static constexpr auto threadPoolSize_ = size_t(10);
+        static constexpr auto queueSize_ = size_t(20);
```

Comment:
I'm curious if these caps are informed by something in the spec, some engineering decisions, etc.

If you believe it's a good idea to have a capped number on these may be good to reflect to other implementations as well.

WDYT?